ci-operator/step-registry/openshift/e2e/openstack/csi/cinder/openshift-e2e-openstack-csi-cinder-workflow.yaml

```diff
@@ -6,7 +6,7 @@ workflow:
     - ref: storage-obj-save
     - ref: storage-conf-csi-openstack-cinder
     test:
-    - chain: openstack-test-e2e
```

Comment:
should the volumes tests from openstack-test still be executed as well?
https://github.com/openshift/openstack-test/blob/main/test/extended/openstack/volumes.go

Reply:
we could, I'll add it later if you don't mind.